docs/recipes/typescript.md

```diff
@@ -1,29 +1,47 @@
 # TypeScript
 
-Translations: [Español](https://github.com/avajs/ava-docs/blob/master/es_ES/docs/recipes/typescript.md), [Français](https://github.com/avajs/ava-docs/blob/master/fr_FR/docs/recipes/typescript.md), [Italiano](https://github.com/avajs/ava-docs/blob/master/it_IT/docs/recipes/typescript.md), [Русский](https://github.com/avajs/ava-docs/blob/master/ru_RU/docs/recipes/typescript.md), [简体中文](https://github.com/avajs/ava-docs/blob/master/zh_CN/docs/recipes/typescript.md)
+AVA comes bundled with a TypeScript definition file. This allows developers to
+leverage TypeScript for writing tests.
 
-AVA comes bundled with a TypeScript definition file. This allows developers to leverage TypeScript for writing tests.
+This guide assumes you've already set up TypeScript for your project. Note that
+AVA's definition has been tested with version 2.8.3.
 
-This guide assumes you've already set up TypeScript for your project. Note that AVA's definition has been tested with version 2.8.3.
+## Setup
 
-Add a `test` script in the `package.json` file. It will compile the project first and then run AVA.
+Firstly, we recommend keeping a `tsconfig.json` file in your test folder. This
+can extend your normal `tsconfig.json` file, but should include your test files.
+This allows changing typescript configuration for tests, and running tools like
+`tslint` on your test files.
+
+Secondly, you will need to have `ts-node` installed.
+
+And last but not least, configure ava to build typescript files by adding
+this configuration to your `package.json` file.
 
 ```json
-{
-	"scripts": {
-		"test": "tsc && ava"
+	"ava": {
+		"babel": false,
```

Comment:
Hopefully we can land https://github.com/avajs/ava/issues/1836 first, and then this should be removed.